scripts/get-latest-fe-version.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Change the working directory to the script's directory
+cd $(dirname "$0") || exit
+
+# Define your GitHub repository owner and name
+repo_owner="ethereum"
+repo_name="fe"
+
+# Make the API request and store the response in a variable
+response=$(curl -s "https://api.github.com/repos/$repo_owner/$repo_name/releases/latest")
+
+# Check if the request was successful (HTTP status code 200)
+if [[ $? -eq 0 ]]; then
+  # Extract the "name" field from the JSON response
+  name=$(echo "$response" | jq -r ".name")
```

Comment:
Do we need to `apt install jq` in the workflow?

Reply:
The runner comes with `jq` preinstalled.